### DISCUSSION TITLE
Welcome to Waline Discussions!

### DISCUSSION THREAD

**Discussion by lizheming** [walinejs/waline#60] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gingesmallfish** [comment]
你好，博主我在登录Waline的时候为什么还要扫码，就那个两部认证

---
Hello, blogger, why do I have to scan the code when I log in to Waline, just the two authentications

**Comment by Gingesmallfish** [comment]
那怎么能


---
how can that

**Comment by Gingesmallfish** [comment]
用什么扫码


---
what scan code

**Comment by Gingesmallfish** [comment]
![image](https://github.com/walinejs/waline/assets/94774975/8f1a45c6-b794-489f-b8f0-2e87a65accc9)

**Comment by Gingesmallfish** [comment]
就是用Waline评论系统的扫码，

---
Just use the scan code of the Waline comment system,

**Comment by Gingesmallfish** [comment]
我登陆的时候，就要喊你设置

---
When I log in, I will call you to set

**Comment by Gingesmallfish** [comment]
打电话？？？？

---
Call up? ? ? ?

**Comment by Gingesmallfish** [comment]
我用QQ扫码也不行，微信也不行

---
I can’t use QQ to scan the code, nor can wechat

**Comment by Gingesmallfish** [comment]
那用什么扫码

---
What kind of scan code

**Reply by Gingesmallfish** [reply]
是的

---
yes

**Reply by Gingesmallfish** [reply]
![image](https://github.com/walinejs/waline/assets/94774975/5939eb0a-849e-4de9-9665-be6a7938401c)
这个两部设置

---
![image](https://github.com/walinejs/waline/assets/94774975/5939eb0a-849e-4de9-9665-be6a7938401c)
This two set

**Reply by Gingesmallfish** [reply]
That is, after I changed the password above, click "two-step verification" to ask you to set up scanning code

**Comment by Gingesmallfish** [comment]
是的

---
yes
